python_modules/dagster/dagster/_core/blueprints/load_from_yaml.py

```diff
@@ -112,4 +117,6 @@ def load_defs_from_yaml(
         for blueprint in blueprints
     ]
 
-    return BlueprintDefinitions.merge(*def_sets_with_code_references).to_definitions()
+    return BlueprintDefinitions.merge(*def_sets_with_code_references).to_definitions(
```

Comment:
I don't feel strongly, but, to avoid threading down the resources further, this could just be:

```python
BlueprintDefinitions.merge(*def_sets_with_code_references, BlueprintDefinitions(resources=resources or {}))
```

Reply:
ah yeah good call